src/neml2/tensors/Scalar.cxx

```diff
@@ -63,4 +64,20 @@ abs(const Scalar & a)
   return Scalar(torch::abs(a), a.batch_sizes());
 }
 
+namespace math
+{
+Scalar
+sigmoid(const Scalar & a, const Scalar & n)
+{
+  neml_assert_broadcastable_dbg(a, n);
+  return 1.0 / 2.0 * (1.0 + math::tanh(n * a));
+}
+
+Scalar
+pow(const Scalar & a, const Scalar & n)
+{
+  neml_assert_broadcastable_dbg(a, n);
+  return torch::pow(a, n);
+}
```

Comment:
I'm a little surprised that we do not already have this function defined somewhere else.

Reply:
Oh yeh I think for whatever reason it was not compile when I used math::pow(a,n) on the previous version. I took that out, re-compile and re-run the test. Everything still pass now.

On the same note: old version math::pow(x,2.0)  cannot compile (x here is either a parameter or a variable). I normally ended up doing x * x .. but everything compiles fine now.